newIDE/app/src/InAppTutorial/InAppTutorialTooltipDisplayer.js

```diff
@@ -1,40 +1,62 @@
 // @flow
 import * as React from 'react';
+import { Trans } from '@lingui/macro';
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
 import Popper from '@material-ui/core/Popper';
 import Fade from '@material-ui/core/Fade';
 import Typography from '@material-ui/core/Typography';
-import { Column, LargeSpacer } from '../UI/Grid';
+import ButtonBase from '@material-ui/core/ButtonBase';
+
+import { Column, Spacer } from '../UI/Grid';
 import { getDisplayZIndexForHighlighter } from './HTMLUtils';
-import InAppTutorialContext, {
-  type InAppTutorialFormattedTooltip,
-} from './InAppTutorialContext';
+import { type InAppTutorialFormattedTooltip } from './InAppTutorialContext';
+import ChevronArrowBottom from '../UI/CustomSvgIcons/ChevronArrowBottom';
 import useIsElementVisibleInScroll from '../Utils/UseIsElementVisibleInScroll';
-import { makeStyles } from '@material-ui/core/styles';
 import { MarkdownText } from '../UI/MarkdownText';
 import RaisedButton from '../UI/RaisedButton';
 import GDevelopThemeContext from '../UI/Theme/ThemeContext';
-import Paper from '@material-ui/core/Paper';
+import Cross from '../UI/CustomSvgIcons/Cross';
+import { LineStackLayout } from '../UI/Layout';
+import ChevronArrowTop from '../UI/CustomSvgIcons/ChevronArrowTop';
+import { textEllipsisStyle } from '../UI/TextEllipsis';
 
-type Props = {|
-  anchorElement: HTMLElement,
-  tooltip: InAppTutorialFormattedTooltip,
-  buttonLabel?: string,
-|};
+const themeColors = {
```

Comment:
Well since this component is "out of the theme" since it's only depends on the palette type, I considered it was better to have those colors hardcoded since there are not obliged to change if the theme colors are changed.